platform/core/src/utils/loadAndCacheDerivedDisplaySets.js

```diff
@@ -0,0 +1,58 @@
+import studyMetadataManager from './studyMetadataManager';
+
+export default function(referencedDisplaySet, studies) {
```

Comment:
JSDoc? Is this appropriate as a "util"?
What is the shape of these params?

Reply:
JSDoc added with params.